config/options.yml

```diff
@@ -55,4 +55,4 @@ rpm_repository:
       :targets:
         - el8
       :rpms:
-        :manageiq-nightly: !ruby/regexp /.+-14\.\d\.\d-(\d\.\d\.)?\d+\.el.+/
+        :manageiq-nightly: !ruby/regexp /.+-14\.\d\.\d-(\d\.\d\.)?\d{14,}+\.el.+/
```

Comment:
oh this is interesting...I guess this collided between 

"14.1.4-1"
"14.1.0-20211208000053"